FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -476,26 +564,29 @@ void Maslow_::generate_calibration_grid() {
   int gridSizeX = 10;
   int gridSizeY = 9;
   int xSpacing = 175;
-  int ySpacing = 100;
+  int ySpacing = 75;
```

Comment:
This was a hack that I did. We should consider making this dynamic again. I just wanted to get it working and then go from there

Reply:
Yeah was meaning to ask why

Reply:
When I tried to run the version that auto generated the points it was auto-generating incorrect points for me. They weren't centered around the center of the frame, I tried to figure out how to fix it but I got confused and I just wanted to make something simple work and then add complexity